docs/visibility.md

```diff
@@ -34,56 +34,62 @@ Elasticsearch takes on the Visibility request load, relieving potential performa
+- **=, !=, >, >=, <, <=**
+- **AND, OR, ()**
+- **BETWEEN ... AND**
+- **IN**
+- **ORDER BY**
 
-- The following operators are supported in List Filters:
+The **ORDER BY** operator is supported only when Elasticsearch is used as the Visibility store.
 
-  - **=, !=, >, >=, <, <=**
-  - **AND, OR, ()**
-  - **BETWEEN ... AND**
-  - **IN**
-  - **ORDER BY**
-  - **LIKE**
+### Partial string match
 
-- Wildcards ('\*', '%', etc.) are supported for string-type List Filter Search Attributes.
-  Use wildcards with the **LIKE** operator to query possible values:
+The `=` operator works like **CONTAINS** to find Workflows with Search Attributes that contain a specific word.
 
-  ```bash
-  // Create a "ProductId" custom Search Attribute of type String
-  tctl admin cluster add-search-attributes --name ProductId --type String
-  // Match values that start with "book"
-  ProductId LIKE "book%"
-  // Match values that contain "favorite"
-  ProductId LIKE "%favorite%"
-  ```
+<!-- note: advanced vis features will be supported in SQL upon the release of v1.20.-->
 
-- A List Filter applies to a single Namespace.
+For example, if you have a Search Attribute `Description` of `Text` type with the value of "The quick brown fox jumps over the lazy dog", searching for `Description=quick` or `Description=fox` will successfully return the Workflow.
+However, partial word searches such as `Description=qui` or `Description=laz` will not return the Workflow.
+This is because [Elasticsearch's tokenizer](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-standard-tokenizer.html) is configured to return complete words as tokens.
 
-- The range of a List Filter timestamp (StartTime, CloseTime, ExecutionTime) cannot exceed 9223372036854775807 (that is, maxInt64 - 1001).
 
-- A List Filter that uses a time range has a resolution of 1 ms on Elasticsearch 6 and 1 ns on Elasticsearch 7.
+### Efficient API usage
 
-- List Filter Search Attribute names are case sensitive.
+An Advanced List Filter API may take longer to respond if it is retrieving a large number of Workflow Executions (over 10,000).
 
-- An Advanced List Filter API may take longer than expected if it is retrieving a large number of Workflow Executions (more than 10 million).
+<!-- Use the `CountWorkflow` API to efficiently count the number of <a class="tdlp" href="/workflows#workflow-execution">Workflow Executions<span class="tdlpiw"><img src="/img/link-preview-icon.svg" alt="Link preview icon" /></span><span class="tdlpc"><span class="tdlppt">What is a Workflow Execution?</span><br /><br /><span class="tdlppd">A Temporal Workflow Execution is a durable, scalable, reliable, and reactive function execution. It is the main unit of execution of a Temporal Application.</span><span class="tdlplm"><br /><br /><a class="tdlplma" href="/workflows#workflow-execution">Learn more</a></span></span></a>. -->
```

Comment:
ditto

Reply:
Generated document. No need to comment on this one since the changes are made in the source file above.